e2e/src/usecases/Send.js

```diff
@@ -154,4 +159,226 @@ export default Send = () => {
+      await element(by.id('SendEnterAmount/Input')).replaceText('0.01')
+      await element(by.id('SendEnterAmount/Input')).tapReturnKey()
+      await waitForElementByIdAndTap('SendEnterAmount/ReviewButton', 30 * 1000)
+      let amount = await element(by.id('SendAmount')).getAttributes()
+      jestExpect(amount.text).toEqual('0.01 cEUR')
+      let emptyComment = await element(by.id('commentInput/send')).getAttributes()
+      jestExpect(emptyComment.text).toEqual('')
+      await addComment(randomComment)
+      let comment = await element(by.id('commentInput/send')).getAttributes()
+      jestExpect(comment.text).toEqual(randomComment)
+    })
+
+    it('Then should be able to send', async () => {
+      await waitForElementByIdAndTap('ConfirmButton', 30 * 1000)
+      await enterPinUiIfNecessary()
+      await expect(element(by.id('errorBanner'))).not.toBeVisible()
+      await waitForElementId('HomeAction-Send', 30 * 1000)
+    })
+
+    it.todo('Then should display transaction as pending')
+
+    it.todo('Then should display transaction as confirmed')
+
+    it.todo('Then should display correct transaction details')
+  })
+
+  describe('When multi-token send flow to recent recipient (new flow)', () => {
+    beforeAll(async () => {
+      await launchApp({
+        newInstance: false,
+        launchArgs: { statsigGateOverrides: `use_new_send_flow=true,use_viem_for_send=true` },
+      })
+    })
+
+    it('Then should navigate to send search input from home action', async () => {
+      await waitForElementByIdAndTap('HomeAction-Send', 30 * 1000)
+      await waitFor(element(by.text('0xe5f5...8846')))
+        .toBeVisible()
+        .withTimeout(10 * 1000)
+    })
+
+    it('Then should be able to click on recent recipient', async () => {
+      await element(by.text('0xe5f5...8846')).atIndex(0).tap()
+      await waitForElementId('SendEnterAmount/Input', 30 * 1000)
+    })
+
+    it('Then should be able to choose token', async () => {
+      await element(by.id('SendEnterAmount/TokenSelect')).tap()
+      await element(by.id('cEURSymbol')).tap()
+      await expect(element(by.text('cEUR')).atIndex(0)).toBeVisible()
+    })
+
+    it('Then should be able to enter amount and navigate to review screen', async () => {
+      await waitForElementByIdAndTap('SendEnterAmount/Input', 30 * 1000)
+      await element(by.id('SendEnterAmount/Input')).replaceText('0.01')
+      await element(by.id('SendEnterAmount/Input')).tapReturnKey()
+      await waitForElementByIdAndTap('SendEnterAmount/ReviewButton', 30 * 1000)
+      await isElementVisible('ConfirmButton')
+    })
+
+    it('Then should display correct recipient', async () => {
+      await expect(element(by.text('0xe5f5...8846'))).toBeVisible()
+    })
+
+    it('Then should be able to add a comment', async () => {
+      await addComment(randomComment)
+      let comment = await element(by.id('commentInput/send')).getAttributes()
+      jestExpect(comment.text).toEqual(randomComment)
```

Comment:
no random tests!

Reply:
This was originally used to verify that the comment showed in the transaction feed; however, there were issues with the most recent transaction not guaranteed to be from this test run and that check was removed. With more recent changes to [getAttributes()](https://wix.github.io/Detox/docs/api/actions/#getattributes) it might be possible to add this check back in.